core-bundle/contao/templates/elements/ce_vimeo.html5

```diff
@@ -1,3 +1,4 @@
+<?php $this->addCspSource('frame-src', $this->src); ?>
```

Comment:
This shouldn‘t be at the top. Maybe we can add it at the bottom like we did with the schema.org data?

Reply:
I have moved to the bottom of the templates now.